### DISCUSSION TITLE
Asset Naming

### DISCUSSION THREAD

**Discussion by EnnzoKim** [Favor-Gift-Reminder/Favor-iOS#33] [Ideas]
우리 이제 각각 브랜치에서 UI를 구현할 때마다
에셋을 넣는 경우가 빈번해질 것 같은데,

에셋 관련 네이밍을 조금 정해보는 게 어떤가 싶어!
예를들어서 나 같은 경우는

![keyboard_arrow_right@3x](https://user-images.githubusercontent.com/97531269/213654185-042dd12e-525f-404a-ab5c-e1a1db96db88.png)
> ic_rightArrow

위와 같이 SFSymbol 같이 직관적으로 네이밍을 해주는 편이야

한번 이야기 해보자~

**Comment by nomatterjun** [comment]
아이콘이라 앞에 `ic_` 붙이고
피그마에 있는 아이콘 이름 그대로 가져온거지?

좋아좋아
Assets 안에 `Icons`처럼 디렉토리도 만들어서 구분해주면 좋을 것 같아.

앞에 붙이는 이름이랑 디렉토리 이름은
에셋 종류가 어떤게 들어갈 지 모르겠어서 추가할때마다 정하기로 할까?

**Reply by EnnzoKim** [reply]
오케이!!

**Comment by nomatterjun** [comment]
나도 에셋 추가해서 작업하려고 하는데

![스크린샷 2023-01-20 22 18 39](https://user-images.githubusercontent.com/60438045/213704006-e69a5793-e08c-4e25-8403-0a322ac47c04.png)

피그마 아이콘 export 할 때 옵션이랑

![스크린샷 2023-01-20 22 18 21](https://user-images.githubusercontent.com/60438045/213703959-09a377c2-5cf8-49d8-aa41-f21f2033d8ae.png)

아이콘 선택할 떄 Wire_Icon_화살표1로 선택해서 했는지
그 안에 있는 keyboard_arrow_up으로 했는지 알려주라..!

**Reply by EnnzoKim** [reply]
전자로 했어 !

**Reply by nomatterjun** [reply]
오케..!

**Comment by nomatterjun** [comment]
지금 발견한건데..!
혹시 아이콘을 svg 파일이 아니라 png 파일로 추출한 이유가 있을까!?

벡터 이미지로 사용하면 좋을 것 같아서..!

**Reply by EnnzoKim** [reply]
아 사실 별 이유는 없어 ㅎㅎ
svg로 사용하면 더 좋구나!! 
이제부터 svg파일로 사용하자!

**Reply by nomatterjun** [reply]
아하..!
svg 포맷은 벡터 이미지라서 해상도가 달라져도 계단 현상이 안발생하거든..!

그럼 svg 파일로 작업할게!

**Reply by nomatterjun** [reply]
아 XCode는 pdf 포맷이 더 잘 어울린다고 들어서
pdf 포맷으로 하면 될 것 같아..!

**Reply by nomatterjun** [reply]
내가 제대로 알아봤는데..

pdf 파일은 ios8.0부터 지원하기 시작했고 svg 파일은 ios13.0부터 지원하기 시작해서
그 이전 기기들을 지원하려고 pdf 포맷을 사용했던거래..
장단점은 없는거나 마찬가지고 svg 포맷이 pdf 포맷보다 용량이 적다고 하네!

ios13.0이면 이제는 그냥 svg 포맷으로 사용해도 될 듯..! 😋

![스크린샷 2023-01-21 00 13 18](https://user-images.githubusercontent.com/60438045/213733053-1a0a57ea-8a90-4ce5-89d4-d2289e081bee.png)

Asset 속성에서
Preserve Vector Data랑
Scale ➡️ Single Scale로 해두고 사용하면 될 것 같아!

**Reply by EnnzoKim** [reply]
오케이 땡큐 !!
